test/e2e/tests/test_grant.py

```diff
@@ -0,0 +1,114 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Integration tests for the KMS Grant resource
+"""
+
+import boto3
+import logging
+import pytest
+import time
+
+from acktest.aws.identity import get_account_id
+from acktest.k8s import resource as k8s
+from acktest.k8s.condition import CONDITION_TYPE_RESOURCE_SYNCED, CONDITION_TYPE_TERMINAL
+from acktest.resources import random_suffix_name
+from e2e import service_marker, CRD_GROUP, CRD_VERSION, load_kms_resource
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.tests.helper import KMSValidator
+
+CREATE_WAIT_AFTER_SECONDS = 30
+MODIFY_WAIT_AFTER_SECONDS = 30
+DELETE_WAIT_AFTER_SECONDS = 30
+DELETE_WAIT_PERIODS = 3
+DELETE_WAIT_PERIOD_LENGTH_SECONDS = 10
+
+GRANT_RESOURCE_PLURAL = "grants"
+
+kms_validator = KMSValidator(boto3.client('kms'))
+
+@pytest.fixture
+def simple_key(kms_client):
+    key = kms_client.create_key()
+
+    yield key
+
+    kms_client.schedule_key_deletion(KeyId=key['KeyMetadata']['KeyId'])
+
+@pytest.fixture
+def simple_grant(simple_key):
+    grant_name = random_suffix_name("simple-grant", 32)
+
+    replacements = REPLACEMENT_VALUES.copy()
+    replacements["GRANT_NAME"] = grant_name
+    replacements["KEY_ID"] = simple_key['KeyMetadata']['KeyId']
+    replacements["ACCOUNT_ID"] = get_account_id()
```

Comment:
Account ID is provided as a default for all replacements - https://github.com/aws-controllers-k8s/test-infra/blob/main/src/acktest/resources.py#L31